src/Main.hs

```diff
@@ -1,6 +1,8 @@
 {-# LANGUAGE DeriveAnyClass #-}
 {-# LANGUAGE UndecidableInstances #-}
 
+-- Color Palette => Rose 300 / Cyan 900 + Dark
```

Comment:
Let's move this to either as as `[ ]` in README.md or close to the render function (eg: `renderBody`)